### DISCUSSION TITLE
Github SMS 2FA keeps sending messages, day and night

### DISCUSSION THREAD

**Discussion by stcksmsh** [community/community#121356] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I recently set up github 2FA sms. During the setup process Github sent me a one time use code to confirm my phone number. I succesfully set up SMS 2FA and thought that was the end of that. Over the last almost 2 days I have received 7 separate authentication setup codes. They are sent every 6 hours, precisely. What is happening?

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it

So, try to check if you have only one Github account. Unfortunately a colleague subscribed 3 times the same account in his Authenticator. The solution was Edit the Authenticator app on Github and repeat the proccess the create 2FA. But to do it, was necessary use the secrets words


If I supported you consider to mark my answer. Good Luck!
